--- individual_old.js	2019-03-05 02:26:38.000000000 -0600
+++ individual_new.js	2019-03-05 02:25:48.000000000 -0600
@@ -5,7 +5,7 @@
 module.exports.validator_patch = function(individual) {
 
     individual.prototype.validatorSchema = Joi.object().keys({
-        name: Joi.string()
+        name: Joi.string().alphanum()
     });
 
     individual.prototype.validatorForCreate = function(record) {
@@ -22,8 +22,9 @@
 
     individual.prototype.validatorForDelete = function(model) {
 
-        //TODO: on the input you have the model to be deleted, no generic Joi checks are available
-
+        if(model.name == 'Undeletable'){
+            return new Error('Model with the name "Undeletable" can not de deleted');
+        }
     };
 
     return individual;
