--- ./validations/individual.js 2019-08-19 23:43:34.000000000 +0000
+++ ./validations/individual_mod.js     2019-08-19 23:47:20.117693361 +0000
@@ -9,7 +9,8 @@
         "$async": true,
         "properties": {
             "name": {
-                "type": "string"
+                "type": "string",
+                "pattern": "^[a-zA-Z0-9]+$"
             }
         }
     }
@@ -32,14 +33,12 @@

     individual.prototype.validateForDelete = async function(record) {

-        //TODO: on the input you have the record to be deleted, no generic
-        // validation checks are available.
-
-        return {
-            error: null
+        if (record.name === 'Undeletable') {
+          throw new Ajv.ValidationError('Record with name "Undeletable" can\'t be deleted')
         }

     }

     return individual
-}
\ No newline at end of file
+}
+

