--- individual_orig.js	2019-03-05 16:54:26.574921039 -0600
+++ individual_patched.js	2019-03-05 16:36:42.470829198 -0600
@@ -5,7 +5,7 @@
 module.exports.validator_patch = function(individual) {
 
     individual.prototype.validatorSchema = Joi.object().keys({
-        name: Joi.string()
+        name: Joi.string().alphanum()
     }).options({
         allowUnknown: true
     });
@@ -24,12 +24,12 @@
 
     individual.prototype.validateForDelete = function(model) {
 
-        //TODO: on the input you have the model to be deleted, no generic Joi checks are available
-
-        return {
-            error: null
-        };
-
+        if(model.dataValues.name === 'Undeletable'){
+            return {error: new Error('Record with name "Undeletable" can\'t be deleted')};
+        }else{
+            return {error: null};
+        }
+        
     };
 
     return individual;
