--- individual_from.js	2019-02-25 22:33:21.000000000 -0600
+++ individual_to.js	2019-02-25 22:50:19.726654518 -0600
@@ -13,7 +13,10 @@
     var individual = sequelize.define('individual', {
 
         name: {
-            type: Sequelize.STRING
+            type: Sequelize.STRING,
+            validate: {
+                isAlpha: true
+            }
         }
     });
 
@@ -25,4 +28,4 @@
     };
 
     return individual;
-};
\ No newline at end of file
+};
