--- aminoacidsequence.js	2019-04-20 18:46:29.000000000 -0500
+++ aminoacidsequence_patch.js	2019-04-21 15:10:42.714230161 -0500
@@ -1,6 +1,8 @@
 const aminoacidsequence = require('../models-webservice/aminoacidsequence');
 const searchArg = require('../utils/search-argument');
 const resolvers = require('./index');
+const request = require('sync-request');
+const baseUrl = 'http://gql_ncbi_sim_srv:3344/aminoAcidSequence/';



@@ -113,10 +115,25 @@
         order,
         pagination
     }, context) {
-        /*
-        YOUR CODE GOES HERE
-        */
-        throw new Error('aminoacidsequences is not implemented');
+
+        let ids = [69905, 10644, 63165];
+        let unfiltered = [];
+
+        let cnt = 0;
+        for(idx in ids){
+            if(idx >= pagination.offset) {
+                console.log(ids[idx]);
+                let res = request('get', baseUrl + ids[idx]);
+                if (res.statusCode !== 200) throw new Error(`Remote service response status code for ${ids[idx]}: ${res.statusCode}`);
+                unfiltered.push(new aminoacidsequence(JSON.parse(res.body.toString('utf8'))));
+
+                cnt++;
+                if(cnt === pagination.limit)
+                    break;
+            }
+        }
+
+        return unfiltered;
     },

     /**
@@ -248,4 +264,4 @@
         */
         throw new Error('csvTableTemplateAminoacidsequence is not implemented');
     }
-}
\ No newline at end of file
+}
