--- aminoacidsequence.js	2019-02-07 01:36:28.000000000 -0600
+++ aminoacidsequence_new.js	2019-02-07 00:07:02.000000000 -0600
@@ -1,6 +1,8 @@
 const aminoacidsequence = require('../models-webservice/aminoacidsequence');
 const searchArg = require('../utils/search-argument');
 const resolvers = require('./index');
+const request = require('sync-request');
+const baseUrl = 'http://gql_ncbi_sim_srv:3344/aminoAcidSequence/';
 
 
 
@@ -88,10 +90,9 @@
     readOneAminoacidsequence: function({
         id
     }, context) {
-        /*
-        YOUR CODE GOES HERE
-        */
-        throw new Error('readOneAminoacidsequence is not implemented');
+        let res = request('get', baseUrl + id);
+        if(res.statusCode !== 200) throw new Error(`Remote service response status code: ${res.statusCode}`);
+        return JSON.parse(res.body.toString('utf8'));
     },
 
     countAminoacidsequences: function({
@@ -102,4 +103,4 @@
         */
         throw new Error('countAminoacidsequences is not implemented');
     }
-}
\ No newline at end of file
+}
